Makefile

```diff
@@ -47,6 +48,7 @@ examples:
 	PLATFORM=nrf52 cargo build --release --target=thumbv7em-none-eabi --example panic --features=custom_panic_handler,custom_alloc_error_handler
 	PLATFORM=nrf52 cargo build --release --target=thumbv7em-none-eabi --example alloc_error --features=alloc,custom_alloc_error_handler
 	PLATFORM=opentitan cargo build --release --target=riscv32imc-unknown-none-elf --examples # Important: This is testing a platform without atomics support
+	PLATFORM=imxrt1050 cargo build --release --target=thumbv7em-none-eabi --examples --features=alloc
```

Comment:
I'm not sure if this should be added to the `examples` target. This is mainly used for testing

Reply:
I removed it now.